pkg/handler-launcher-com/cmd/v1/cmd.proto

```diff
@@ -158,3 +159,13 @@ message FreezeRequest {
   VMI vmi = 1;
   int32 unfreezeTimeoutSeconds = 2;
 }
+
+message MemoryDumpRequest {
+  VMI vmi = 1;
+  string dumpPath = 2;
+}
+
+message MemoryDumpResponse {
+  Response response = 1;
+  bool completed = 2;
```

Comment:
I dont understand the question. Im using the Response like all the other Responses but add a bool that I wanted to indicate if the memory dump completed or not, in the Response there is bool for success and a string message. success is not the same as completed as the request for the dump can succeed since it was triggered successfully and after that the command will return completed if the async request completed..

Reply:
All I said is that bool is limited (you will rarely see it here, or in responses in general) and you may later need a response which is more informative than false.

Reply:
I see, for completion its a pretty much yes or no question, and I do updated the message in the Response part if I had some error and that maybe why the completion is false.. so I think that coveres it..